libraries/standard/http/include/http_client.h

```diff
@@ -84,7 +84,7 @@ typedef struct HTTPNetworkContext HTTPNetworkContext_t;
 /**
  * @brief Transport interface for sending data over the network.
  *
- * If the number of bytes written returned is less than bytesToWrite, then
+ * If the number of bytes written returned is not equal to bytesToWrite, then
  * #HTTPClient_Send will return HTTP_NETWORK_ERROR. If a negative value is
```

Comment:
(Nit): `#HTTP_NETWORK_ERROR` for doxygen to highlight the error code.

Reply:
Thanks